### PR TITLE
Monitoring apps logs 1.0.1

### PR DESCRIPTION
**Pull request by imranMnts**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by imranMnts** [comment]
Hello @imrekel 

Can you review my PR or if I can have any status about that, please.

File: steps/monitoring-apps-logs/1.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+title: Mobile apps logs
+summary: |
+  Check if your Android app kept any logs in your release build
+description: "Check if an Android application contain any logs in the release app\nIf
+  yes, this step will fail and generate a failure where you can see the count \n\n###
+  Troubleshooting\n\n- Be aware, you have to launch this step after [apps-decompiler](https://github.com/imranMnts/bitrise-step-apps-decompiler)
+  to have informations from your APK\n"
```
Comment (koral--) [inline steps/monitoring-apps-logs/1.0.1/step.yml:7]: Why trailing newline?

File: steps/monitoring-apps-logs/1.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+website: https://github.com/odemolliens/bitrise-step-monitoring-apps-permissions
```
Comment (koral--) [inline steps/monitoring-apps-logs/1.0.1/step.yml:8]: How permissions monitoring is related to logs monitoring?

File: steps/monitoring-apps-logs/1.0.1/step.yml (hunk 1 above)
Comment (imranMnts) [reply steps/monitoring-apps-logs/1.0.1/step.yml:7]: I don't know
I don't have it in my local repository, may be added during `bitrise share`?

Screenshot 2021-04-06 at 14 56 00

File: steps/monitoring-apps-logs/1.0.1/step.yml (hunk 2 above)
Comment (imranMnts) [reply steps/monitoring-apps-logs/1.0.1/step.yml:8]: I missed to update these links, I did it in #2977 thanks

**Comment by imranMnts** [comment]
Moved to #2977

**Pull request closed without merging**